application/routes/term.py

```diff
@@ -59,7 +59,7 @@ def generate():
             'port': TERMINAL_PORT,
             'term_id': term.id
         }
-        yield json.dumps(result)
+        yield jsonify(result)
```

Comment:
The response type is `application/octet-stream` here and using `jsonify` introduces unnecessary overhead with mere benefits. Consider reverting the change.